runner.py

```diff
@@ -74,6 +117,12 @@ def load_index():
     # init loading of query redactor
     config.query_redactor  # pylint: disable=W0104
 
+    # store service configuration if enabled
+    if not config.ols_config.user_data_collection.config_disabled:
```

Comment:
@onmete We have the config file in /etc/ols/olsconfig.yaml right , we cant use that as is without this new file ?